eat.go

```diff
@@ -27,7 +37,7 @@ func (e *Eat) FromCBOR(data []byte) error {
 	return dm.Unmarshal(data, e)
 }
 
-// ToCBOR serializes the receiver Eat into CBOR encoded EAT
+//nolint:gocritic // ToCBOR serializes the receiver Eat into CBOR encoded EAT
```

Comment:
To avoid cluttering the doc string with spurious material, please move the lint directive to the following line (here and below).

Reply:
Resolved in [f00bac2](https://github.com/veraison/eat/pull/36/commits/f00bac2ae5e4a18abeca2c73f20e1490536966f8)